### DISCUSSION TITLE
Other languages

### DISCUSSION THREAD

**Discussion by tuxedotje** [standardnotes/forum#3642] [General] [answered]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

i like to have spellcheck in my OWN language.  I cannot find another spell check than english.

**Describe alternatives you've considered**

**Additional context**

**Comment by effieeee** [comment, marked as answer]
Hi @tuxedotje, while I’m afraid the web and mobile apps do not have spell check at the moment, the **desktop app** does have additional spellcheck languages. You’ll find them under the **Edit** dropdown menu (see the screenshots below for both the classic and themed menu bar)

<img width="335" alt="IMG_0627" src="https://github.com/standardnotes/forum/assets/54989748/12a3b379-269c-4dc9-8187-e05c7d48335c">

<img width="355" alt="image" src="https://github.com/standardnotes/forum/assets/54989748/15abc4ef-b028-484b-adc5-21bb0f20332b">


Under this dropdown menu, look for **Spellcheck Languages**, and you should be able to select and deselect your preferred languages from the list 🙏️

**Reply by tuxedotje** [reply]
Thank you very much. We will have to wait for a spellcheck on all devices.
